aplib.net-demo/Assets/Scripts/Entities/RespawnableComponent.cs

```diff
@@ -0,0 +1,65 @@
+using JetBrains.Annotations;
+using System;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace Entities
+{
+    /// <summary>
+    /// Component that allows an entity to respawn by resetting its position and velocity.
+    /// </summary>
+    public class RespawnableComponent : MonoBehaviour
+    {
+        [SerializeField] public Area _spawnArea;
```

Comment:
Voor zover ik weet is dit gebruikelijk voor wat we tot nu toe in de code hebben gedaan. Ook staat er een public maar is de variabele _spawnArea, moet dit private zijn?
```suggestion
        [SerializeField] 
        public Area _spawnArea;
```